client/src/components/admin/student/AddStudent.jsx

```diff
@@ -1,66 +1,186 @@
 import React, { Component } from 'react';
-import { Button, Form, Header } from 'semantic-ui-react';
+import { Button, Form, Header, Input, Message, Icon } from 'semantic-ui-react';
 import PropTypes from 'prop-types';
+import axios from 'axios';
 
-const enrollmentStatus = [
-  { key: 1, text: 'Alum', value: 'alum' },
+const { GHOSTBUSTER_BASE_URL } = process.env;
+
+const enrollmentStatusList = [
+  { key: 0, text: 'Alum', value: 'alum' },
+  { key: 1, text: 'Completed', value: 'completed' },
```

Comment:
You can use `text-transform: capitalize;` in CSS rather than keep a separate data value just for the UI.

Reply:
fixed!